CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ add_library(${PROJECT_NAME}-common
   common/zhash.c
   common/zmaxheap.c)
 target_link_libraries(${PROJECT_NAME}-common pthread m)
+set_target_properties(${PROJECT_NAME}-common PROPERTIES COMPILE_FLAGS "-std=gnu99 -Wall -Wno-unused-parameter -Wno-unused-function -pthread -O4")
```

Comment:
Remove the one at https://github.com/clearpathrobotics/apriltag/pull/1/files#diff-af3b638bc2a3e6c650974192a53c7291R4